lib/makara/logging/subscriber.rb

```diff
@@ -2,12 +2,17 @@ module Makara
   module Logging
 
     module Subscriber
+      IGNORE_PAYLOAD_NAMES = ["SCHEMA", "EXPLAIN"]
 
       def sql(event)
         name = event.payload[:name]
-        name = [current_wrapper_name(event), name].compact.join(' ')
-        event.payload[:name] = name
-        super(event)
+        if IGNORE_PAYLOAD_NAMES.include?(name)
```

Comment:
That's what I originally did, but `IGNORE_PAYLOAD_NAMES` isn't present before ActiveRecord 4 and could disappear at any time, so it's a bit safer to define it ourselves.  I don't feel too strongly either way.